app/utils/Color.ts

```diff
@@ -0,0 +1,25 @@
+import { on } from "events";
+
+const colors =
```

Comment:
Always write classes as nouns with in PascalCase (Capital Letters)

Reply:
Hm, this is no class though, is it? It's a normal variable so I'd say it should remain lowercase? (debatable but not very important)

Reply:
I know that it holds only variable but by definition it's a class or am I missing sth. Anyway...